cnf-certification-test/suite_test.go

```diff
@@ -159,6 +160,7 @@ func TestTest(t *testing.T) {
 
 	// Run tests specs only if not in diagnostic mode, otherwise all TSs would run.
 	if !diagnosticMode {
+		daemonset.PartnerRepoDaemonset()
```

Comment:
I think I'd use another name for this function. Something like `DeployTestDaemonset()` or similar.

Reply:
maybe PartnerDeployTestDamenoset()? @jmontesi

Reply:
I agree.